prime-dotnet-webapi/Configuration/BusinessEventTypeConfiguration.cs

```diff
@@ -17,7 +17,8 @@ public override IEnumerable<BusinessEventType> SeedData
                     new BusinessEventType { Code = 5, Name = "Enrollee", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new BusinessEventType { Code = 6, Name = "Site", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
                     new BusinessEventType { Code = 7, Name = "Admin View", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
-                    new BusinessEventType { Code = 8, Name = "Organization", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE }
+                    new BusinessEventType { Code = 8, Name = "Organization", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE },
+                    new BusinessEventType { Code = 9, Name = "Pharmanet API Call", CreatedTimeStamp = SEEDING_DATE, UpdatedTimeStamp = SEEDING_DATE }
```

Comment:
In the ticket it states "Add new businessEvent Type for PharmaNet query", not sure if "Pharmanet API Call" might be a little too much developer jargon since this is a business facing event. Personally i'm not sure which way to go, just wanted to raise it, in the case the admins can get the most value from these events.

Reply:
Good point @hogmann , so I asked the question in the Jira ticket

Reply:
Nick thinks the wording is OK, see his comment: https://bcgovmoh.atlassian.net/secure/RapidBoard.jspa?rapidView=1&modal=detail&selectedIssue=PRIME-1402